### PR TITLE
Use ddev config to change configuration

### PR DESCRIPTION
**Pull request by sypets**
Also, the file .ddev/config.yaml was mentioned at the beginning of the procedure where it does not exist yet. This was changed.

Also, the command "ddev config" is used to edit the file.

Additionally, the nodejs version is set, which may be necessary, depending on the DDEV version.

### CODE REVIEW

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 1]
```diff
@@ -29,9 +29,8 @@ installation using the cloned TYPO3 CMS Git repository.
 
 You don't need to have a Webserver, a Database or PHP running on your system.
 Everything will be supplied by DDEV. In fact, if you do have a Webserver
-or Database running on your machine, make sure there are no conflicts.
-You can change the default ports DDEV uses (e.g. port 80 / 443 for Webserver)
-in :file:`.ddev/config.yaml` before you start it.
+or Database running on your machine, make sure there are no conflicts (e.g.
+change the ports in the ddev config, see below).
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:33]: I suggest to link "ddev config" to the according section and remove "see below".

Comment (sypets) [reply Documentation/Appendix/SettingUpTypo3Ddev.rst]: Pushed another change. Sorry about late follow-up.

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 2]
```diff
@@ -69,32 +69,30 @@ DDEV should suggest the correct defaults and you just need to press ENTER::
    > Found a typo3 codebase at /var/www/t3coredev.
    > Project Type [backdrop, drupal6, drupal7, drupal8, drupal9, laravel, magento, magento2, php, typo3, wordpress] (typo3):
 
-Edit configuration
-==================
+Change configuration
+====================
 
-Edit the configuration file :file:`.ddev/config.yaml`.
+In order to make further changes to the configuration, use ddev config as shown
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:75]:
```suggestion
In order to make further changes to the configuration, use :ref:`ddev config <ddev-configure>`
```

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 3]
```diff
@@ -69,32 +69,30 @@ DDEV should suggest the correct defaults and you just need to press ENTER::
...
+below or edit the configuration file :file:`.ddev/config.yaml` manually.
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:76]:
```suggestion
or edit the configuration file :file:`.ddev/config.yaml` manually.
```

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 4]
```diff
@@ -163,7 +161,7 @@ now:
       The first command is required once, the second (build) command is required after
       every change of a resource file.
 
-      Be aware that until TYPO3 11.5 yarn was used. 
+      Be aware that until TYPO3 11.5 yarn was used.
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:164]:
```suggestion
      Be aware that until TYPO3 v11.5 yarn was used.
```

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 5]
```diff
@@ -69,32 +69,30 @@ DDEV should suggest the correct defaults and you just need to press ENTER::
    > Found a typo3 codebase at /var/www/t3coredev.
    > Project Type [backdrop, drupal6, drupal7, drupal8, drupal9, laravel, magento, magento2, php, typo3, wordpress] (typo3):
 
-Edit configuration
-==================
+Change configuration
+====================
 
-Edit the configuration file :file:`.ddev/config.yaml`.
+In order to make further changes to the configuration, use :ref:`ddev config <ddev-configure>`
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:75]:
```suggestion
In order to make further changes to the configuration, use ddev config as shown below
```

Sorry, misread :-(

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**